.github/workflows/publish-version.yml

```diff
@@ -143,25 +64,30 @@ jobs:
         env:
           STX_NETWORK: testnet
 
-      - name: Install `app/` packages
-        run: cd app && yarn --frozen-lockfile --ignore-scripts
-        env:
-          STX_NETWORK: testnet
-
       - name: Install `concat-map` - Windows only
         run: yarn add concat-map --ignore-scripts --frozen-lockfile
         if: matrix.os == 'windows-latest'
         env:
           STX_NETWORK: testnet
 
+      - name: Install `app/` packages
+        run: cd app && yarn --frozen-lockfile --ignore-scripts
+        env:
+          STX_NETWORK: testnet
+
+      - name: Extract version
+        id: extract_version
+        uses: Saionaro/extract-package-version@v1.0.6
+
       - name: Build releases
         run: ./node_modules/.bin/cross-env yarn package-${{ matrix.NPM_COMMAND }}
         env:
-          SHA: ${{ github.event.pull_request.head.sha }}
           GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          SHA: ${{ github.event.pull_request.head.sha }}
```

Comment:
If this isn't running from pull requests anymore, this will likely resolve to an empty string.

Reply:
This is just for build info, doesn't hurt, but indeed may need to change